### PR TITLE
Feature/competition page

### PR DESCRIPTION
**Pull request by Pedrocssg**
Competition Page completed, give it a look whenever you can!

### CODE REVIEW

File: src/pages/competicao-programacao.js [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
+import Icon from "../images/svg/competicao.inline.svg"
 
 const CompetitionPage = () => (
   <Layout>
     <SEO title="Competição de programação" />
+    <PageBanner>
+      <Icon />
+    </PageBanner>
     <Competition />
     <Link to="/">Go back to the homepage</Link>
```
Comment (DuarteOliveira8) [inline src/pages/competicao-programacao.js]: You can remove this link.

**Comment by DuarteOliveira8** [comment]
And change the link on the navbar so that we can see the page.

File: src/pages/competicao-programacao.js (hunk 1 above)
Comment (Pedrocssg) [reply src/pages/competicao-programacao.js]: Done!

Comment (Pedrocssg) [reply src/pages/competicao-programacao.js]: Done!

File: src/styles/competition/competition.module.css [hunk 2]
```diff
@@ -0,0 +1,179 @@
+.competitionText {
+  font-size: 1.15em;
+}
+
+.conditions {
+  background-color: var(--color3);
+  padding-bottom: 10%;
```
Comment (smbea) [inline src/styles/competition/competition.module.css]:
```suggestion
  padding-bottom: 12%;
```

File: src/styles/competition/competition.module.css [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+    bottom: 120px;
+  }
+
+  .prizeType {
+    font-size: 2em;
+    margin-left: 0;
+  }
+
+  .prizeDescription {
+    font-size: 1.15em;
```
Comment (smbea) [inline src/styles/competition/competition.module.css]:
```suggestion
    font-size: 16px;
```

File: src/styles/competition/competition.module.css [hunk 4]
```diff
@@ -0,0 +1,179 @@
+.competitionText {
+  font-size: 1.15em;
```
Comment (smbea) [inline src/styles/competition/competition.module.css]:
```suggestion
  font-size: 16px;
```

File: src/styles/competition/competition.module.css [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+  opacity: 1;
+  height: 280px;
+  width: 280px;
+  padding: 15px !important;
+}
+
+.prizeType {
+  margin-top: 15px;
+  margin-left: 15px;
+  font-size: 2em;
```
Comment (smbea) [inline src/styles/competition/competition.module.css]:
```suggestion
  font-size: 1.3em;
```

File: src/styles/competition/competition.module.css [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+  font-weight: 600;
+  text-align: left;
+  letter-spacing: 0.7px;
+  color: #361784;
+}
+
+.prizeDescription {
+  margin-top: 10px;
+  margin-left: 15px;
+  font-size: 1.25em;
```
Comment (smbea) [inline src/styles/competition/competition.module.css]:
```suggestion
  font-size: 16px;
```

File: src/styles/competition/competition.module.css [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+    height: 240px;
+    width: 240px;
+  }
+
+  .prizeSquares {
+    bottom: 120px;
+  }
+
+  .prizeType {
+    font-size: 2em;
```
Comment (smbea) [inline src/styles/competition/competition.module.css]:
```suggestion
    font-size: 1.3em;
```

**Comment by smbea** [comment]
77F0C10D-6155-49F2-A36A-7D824608D31C

Add a bit more space between rules and prizes for mobile please.

**Pull request merged**